pkg/scheduler/framework/parallelize/parallelism.go

```diff
@@ -51,6 +52,12 @@ func chunkSizeFor(n, parallelism int) int {
 }
 
 // Until is a wrapper around workqueue.ParallelizeUntil to use in scheduling algorithms.
-func (p Parallelizer) Until(ctx context.Context, pieces int, doWorkPiece workqueue.DoWorkPieceFunc) {
-	workqueue.ParallelizeUntil(ctx, p.parallelism, pieces, doWorkPiece, workqueue.WithChunkSize(chunkSizeFor(pieces, p.parallelism)))
+func (p Parallelizer) Until(ctx context.Context, pieces int, doWorkPiece workqueue.DoWorkPieceFunc, labelValue string) {
+	doWorkPiesWithMetrics := func(piece int) {
```

Comment:
this should be enough

```suggestion
	withMetrics := func(piece int) {
```